main-page.css

```diff
@@ -0,0 +1,213 @@
+body {
+    margin: 0;
+}
+.right-pannel {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    border-radius: 10px 0 0 10px;
+    width: 80px;
+    height: 250px;
+    background-color: white;
+    position: fixed;
+    float: right;
+    top: 40%;
+    right: 0;
+}
+.icons {
+    width: 50px;
+    height: 50px;
+}
+.icons:first-child {
+    margin-top: 10px;
+}
+.icons:last-child {
+    margin-bottom: 10px;
+}
```

Comment:
Use parent block padding instead of pseudo-classes of children.

Reply:
fixed